thyrel-api/thyrel-api/DataProvider/SessionDataProvider.cs

```diff
@@ -71,12 +73,17 @@ public async Task<Session> GetSessionById(int sessionId)
         /// </summary>
         /// <param name="roomId"></param>
         /// <returns></returns>
-        public async Task<Session> GetCurrentSessionByRoomId(int roomId)
+        public async Task<SessionDto> GetCurrentSessionByRoomId(int roomId)
         {
+            var playerDataProvider = new PlayerDataProvider(_holyDrawDbContext);
+
             var session = await _holyDrawDbContext.Session
                 .OrderBy(s => s.CreatedAt)
                 .LastOrDefaultAsync(s => s.RoomId == roomId && s.FinishAt == null);
```

Comment:
ici il faut directement sélectionné le SessionDto et non le crée par la suite
```suggestion
                .LastOrDefaultAsync(s => s.RoomId == roomId && s.FinishAt == null)
                .Select(s => new SessionDto(s));
```
test si il faut le placer avant ou après le `LastOrDefault`

Reply:
Avant le `LastOrDefault`